jest.config.js

```diff
@@ -1,10 +1,13 @@
 module.exports = {
   collectCoverageFrom: [
+    'src/*.js',
     'src/**/*.{js,jsx}',
+    '!src/*.test.js',
```

Comment:
Is this not already covered by the line beneath?

Reply:
No, `configureStore.js`, for instance, didn't show up in the coverage report.